app/livechat/server/lib/Contacts.js

```diff
@@ -55,7 +55,7 @@ export const Contacts = {
 
 		LivechatVisitors.updateById(contactId, updateUser);
 
-		const rooms = LivechatRooms.findByVisitorId(contactId).fetch();
+		const rooms = LivechatRooms.findOpenByVisitorId(contactId).fetch();
```

Comment:
Only open rooms receive visitor registration?

Reply:
No, but we only care about open rooms when updating contact info. The old query would have updated all rooms, even if they were months old 😬